02_configure_host.sh

```diff
@@ -112,6 +112,12 @@ if [ "$MANAGE_INT_BRIDGE" == "y" ]; then
     fi
 fi
 
+# If there were modifications to the /etc/sysconfig/network-scripts/ifcfg-*
+# files, it is required to enable the network service
+if [ "$MANAGE_INT_BRIDGE" == "y" ] || [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
+  sudo systemctl enable network
```

Comment:
I suspect this was only needed for RHEL8 since we install `network-scripts` for `ifcfg` file support, which isn't there by default.  I suppose this is harmless elsewhere though.